cocos/core/data/utils/attribute-internal.ts

```diff
@@ -1,3 +1,28 @@
+/*
+ Copyright (c) 2013-2016 Chukong Technologies Inc.
+ Copyright (c) 2017-2023 Xiamen Yaji Software Co., Ltd.
```

Comment:
Only `Copyright (c) 2023 Xiamen Yaji Software Co., Ltd.` is enough.

Reply:
Solved